src/Esquio.AspNetCore/Blazor/Feature.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Esquio.Abstractions;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.Extensions.Primitives;
+using System.Threading.Tasks;
+
+namespace Esquio.AspNetCore.Blazor
+{
+    public class Feature : ComponentBase
+    {
+        private bool isEnabled;
+        private static readonly char[] FeatureSeparator = new[] { ',' };
+
+        /// <summary>
+        /// The product name that determines whether the content can be displayed. If you do not specicify a product name,
+        /// default will be used.
+        /// </summary>
+        [Parameter] public string Product { get; set; } = "default";
```

Comment:
EsquioConstants.cs define a public constant for default product, can you use it?

https://github.com/Xabaril/Esquio/blob/aa7d3ab4f8703f1f1d2d35d41f297a1a92c0cc46/src/Esquio/EsquioConstants.cs#L7

Reply:
Done!